tests/4.5/target/test_target_private.c

```diff
@@ -35,26 +33,22 @@ int main() {
       compute_array[i][j] = 0;
 
 
-  omp_set_num_threads(4);
-#pragma omp parallel
-{
-  int fp_val = omp_get_thread_num();
-  int p_val=0;
-
-  real_num_threads = omp_get_num_threads();  
+    
 
-#pragma omp target map(tofrom:compute_array) firstprivate(fp_val) private(p_val)
+#pragma omp target map(tofrom:compute_array, real_num_threads) private(p_val)
```

Comment:
Just for reading the code, how about `map(from:real_num_threads)`? (Admittedly, it mainly helps when reading the diff – but still, its value is undefined before the `omp target`.)

Reply:
@tob2, yes "from" would be adequate..just being lazy.